libnavigation-base/build.gradle

```diff
@@ -15,6 +15,8 @@ android {
 }
 
 dependencies {
+    testImplementation project(':libtesting')
```

Comment:
NIT Could you put it down the block? Same thing applies to the rest of the modules

Reply:
done